torchinductor/codegen/triton.py

```diff
@@ -78,6 +78,19 @@ def log(x):
     def relu(x):
         return ops.maximum("0", x)
 
+    @staticmethod
+    def round(x):
+        return f"tl.where({x}<0, {x}-0.5, {x}+0.5).to(tl.int32).to(tl.float32)"
```

Comment:
Nothing out of box that I can think of.

Reply:
Let's create an issue to track it, and fix it later.  I believe Triton is working on adding support for calling cuda library functions, and stuff like this will become easier at that point.

Reply:
For future reference, here's what nvcc does https://godbolt.org/z/o7henjE9o, the difference is `add.rz.ftz` generated for `x+/-0.5` instead of `add.ftz` that's generated for the regular addition

Reply:
Ah, interesting.   Any idea how to make Triton generate `add.rz.ftz`?

Reply:
Not yet